test/e2e/tests/errors.spec.js

```diff
@@ -48,7 +49,8 @@ describe('Sentry errors', function () {
           return isPending === false;
         }, 10000);
         const [mockedRequest] = await mockedEndpoint.getSeenRequests();
-        const mockJsonBody = mockedRequest.body.json;
+        const mockTextBody = mockedRequest.body.text.split('\n');
```

Comment:
It seems the body of the request is now a `string`, but one that is split into three parts.